src/components/DataGrid/utils.tsx

```diff
@@ -0,0 +1,149 @@
+import { GridActionsCellItem, GridColDef, GridLocaleText } from '@mui/x-data-grid';
+import {
+  CreateGridColDefFunction,
+  CreateSingleGridColDefFunction,
+  InformatieObjectT,
+  ZaaktypeT,
+} from '../../types/types';
+import text from '../../utils/text';
+import array from '../../utils/array';
+import extract from '../../utils/extract';
+import { ArrowForward } from '@mui/icons-material';
+import { Tooltip } from '@mui/material';
+
+/**
+ * Default filters for the DataGrid
+ * These filters can't be set to hidden
+ */
+export const defaultFilters = ['omschrijving', 'identificatie'];
+
+/**
+ * Locale text for the DataGrid
+ * @see https://mui.com/components/data-grid/localization/
+ */
+
+export const dataGridLocaleText: Partial<GridLocaleText> = {
+  toolbarDensity: 'Rijhoogte',
+  toolbarDensityCompact: 'Compact',
+  toolbarDensityStandard: 'Standaard',
+  toolbarDensityComfortable: 'Comfortabel',
+  toolbarColumns: 'Kolommen',
+  toolbarColumnsLabel: 'Kolommen',
+  columnsPanelTextFieldLabel: 'Zoek kolommen',
+  columnsPanelTextFieldPlaceholder: 'Titel kolom',
+  columnsPanelDragIconLabel: 'Kolom verplaatsen',
+  columnsPanelHideAllButton: 'Alles verbergen',
+  columnsPanelShowAllButton: 'Alles tonen',
+};
+
+/**
+ * Add an id to each row
+ * @param rows Zaaktypen or Attributen
+ * @returns rows with id
+ */
+export const rowsWithId = (rows: ZaaktypeT[] | InformatieObjectT[]) =>
+  rows.map((row, i) => {
+    return { ...row, id: row?.url ? extract.uuid(row.url) : i };
```

Comment:
The zaaktypen should always have a URL (so I think this check is not needed and you can always use the Uuid)

Reply:
Discussed: this is needed for when the data hasn't been loaded yet.